vslib/SwitchState.cpp

```diff
@@ -17,6 +17,25 @@ using namespace saivs;
 
 #define VS_COUNTERS_COUNT_MSB (0x80000000)
 
+const std::map<sai_stat_id_t, std::string> SwitchState::createStatIdMap()
+{
+    SWSS_LOG_ENTER();
+
+    std::map<sai_stat_id_t, std::string> statIdMap = {
+        { SAI_PORT_STAT_IF_IN_OCTETS, "rx_bytes" },
+        { SAI_PORT_STAT_IF_IN_UCAST_PKTS, "rx_packets" },
+        { SAI_PORT_STAT_IF_IN_ERRORS, "rx_errors" },
+        { SAI_PORT_STAT_IF_IN_DISCARDS, "rx_dropped" },
+        { SAI_PORT_STAT_IF_OUT_OCTETS, "tx_bytes" },
+        { SAI_PORT_STAT_IF_OUT_UCAST_PKTS, "tx_packets" },
+        { SAI_PORT_STAT_IF_OUT_ERRORS, "tx_errors" },
+        { SAI_PORT_STAT_IF_OUT_DISCARDS, "tx_dropped" }
+    };
+
+    return statIdMap;
+}
+const std::map<sai_stat_id_t, std::string> SwitchState::m_statIdMap = SwitchState::createStatIdMap();
```

Comment:
i thought you will assign this map directly but if you want to do it as function, so let it be